src/rgw/rgw_reshard.cc

```diff
@@ -554,6 +554,7 @@ int RGWBucketReshard::do_reshard(int num_shards,
       }
 
       for (auto iter = entries.begin(); iter != entries.end(); ++iter) {
+	sleep(32);
```

Comment:
did you plan to just remove this, or were you thinking of creating some way to trigger fault injection?

Reply:
Yeah, two of the commits are for testing purposes only. They'll be stripped away when I remove DNM.